content/blog/open-sourcing-workshops-on-demand-part-5-create-a-workshop.md

```diff
@@ -0,0 +1,226 @@
+
+A﻿s an admin of the Workshops-on-demand infrastructure, I had to perform several tasks:
+
+##### O﻿n the backend server:
+
+1. ###### Test and validate installation of the new kernel on the staging backend server by:
+
+* * Creating a new branch for this test
+
+  * M﻿odifying the [backend server installation yaml file ](https://github.com/Workshops-on-Demand/wod-backend/blob/main/ansible/install_backend.yml#L326)to include the new kernel.
+
+  * Validating the changes by testing a new backend install process.
+
+  * Pushing the changes to the github repo.
+
+2. ###### Creating a user for the workshop developer on the test/dev and staging backend servers.
+3. ###### Providing to the developer the necessary information to connect to the test/dev and staging backend servers.
+4. ###### Copy over the workshop developer's home folder a workshop template containing examples of introduction, conclusion and lab notebooks, allowing him to start his work.
+5. **Give the developer the wod-notebook repo url for him to fork the repo and work locally on his machine (when workshop does not require appliance but just a jupyter kernel for instance)**
+6. **Whenever ready, a pull request can be made. The admin can then review and accept it. The admin can then perform the necessary steps to prepare the infrastructure to host the workshop**
+
+
+
+##### O﻿n the database server:
+
+In order to exist, a workshop requires serveral entries in the database:
+
+I﻿n the **Workshops table:**
+
+A﻿ new entry will need the following:
+
+![](/img/wod-db-go-1.png "Workshop's fields in the Database.")
+
+* **An id:** A workshop id to be used by backend server automation and Replays table to reference the associated replay video of the workshop.
+* **A﻿ name:** The workshop's name as it will will be displayed on the registration portal.
+* **A name of the folder** containing all the workshop's notebooks
+* **A﻿ description / abstract**
+* **A﻿ capacity:** The number of maximum concurrent students allowed to take on the workshop.
+* **A﻿ student range:** The range between which students get picked at registration time.
+* **R﻿eset and ldap** entries are to be used by backend server automation if dedicated reset scripts and ldap authentication are required by the workshop.
+* **A﻿ session type:** Workshops-on-Demand by default
+* **A﻿ location:** If your setup includes multiple production sites, use this field to allocate workshops according to your needs. In the case of the HPE Developer Community, some workshops can only run on a HPE GreenLake cloud environment. As a consequence, the location is set to greenlake in this case.
```

Comment:
Can you use the proper spelling for GreenLake in the last part of this sentence or does it reflect a comman/file name that's all in lower case?